core/src/io/anuke/mindustry/core/NetClient.java

```diff
@@ -198,6 +199,16 @@ public static String colorizeName(int id, String name){
         return "[#" + player.color.toString().toUpperCase() + "]" + name;
     }
 
+    @Remote(called = Loc.client, variants = Variant.one)
+    public static void onConnect(String ip, int port){
+        netClient.disconnectQuietly();
+        state.set(State.menu);
+        logic.reset();
+
+        Vars.netClient.beginConnecting();
+        net.connect(ip, port, Actions::new);
```

Comment:
That's pretty much the same thing. `Actions` is a class for animated UI elements, it has nothing to do with this connections callback. Don't use it.

What you want is an empty callback, which would be `() -> {}`.